app/vmctl/flags.go

```diff
@@ -373,6 +375,12 @@ var (
 			Usage:   "VictoriaMetrics password for basic auth",
 			EnvVars: []string{"VM_NATIVE_SRC_PASSWORD"},
 		},
+		&cli.StringFlag{
+			Name: vmNativeSrcHeaders,
+			Usage: "Optional HTTP headers to send with each request to the corresponding source address. \n" +
+				"For example, --vm-native-src-headers='My-Auth:foobar' would send 'My-Auth: foobar' HTTP header with every request to the corresponding source address. \n" +
+				"Multiple headers must be delimited by '^^': --vm-native-src-headers='header1:value1^^header2:value2'",
```

Comment:
This follows the convention established by the `remote-read-headers` flag:
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/a0cbef1c4696595beb619ff6e3118efa447cd2d7/app/vmctl/flags.go#L501